lib/mini_scheduler/manager.rb

```diff
@@ -92,7 +92,7 @@ def process_queue
 
           klass.new.perform
         rescue => e
-          MiniScheduler.handle_job_exception(e, message: "Running a scheduled job", job: klass)
+          MiniScheduler.handle_job_exception(e, message: "Running a scheduled job", job: { "class" => klass })
```

Comment:
Looking at the Sidekiq source ([`Sidekiq::Processor#process`](https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/processor.rb#L145)), `:job` is most likely the job description (i.e. the parsed `jobstr` from Redis).

It doesn't look like `#dispatch` or `#invoke` change the hash, so `job["class"]` is most likely a string. Should I reflect that here as well, i.e.

```ruby
MiniScheduler.handle_job_exception(e, message: "...", job: { "class" => klass.to_s })
```

For my use case, `sentry-raven`, this would not matter much (because it constructs `"Sidekiq/#{context[:job]["class"]}"` which works for both String and Class instances), but it diverges from the calling convention used by Sidekiq.